mlflow/tracking/_model_registry/utils.py

```diff
@@ -47,6 +47,28 @@ def set_registry_uri(uri):
                   Databricks CLI
                   `profile <https://github.com/databricks/databricks-cli#installation>`_,
                   "databricks://<profileName>".
+
+    .. code-block:: python
+        :caption: Example
+
+        import mflow
+
+        # Set model registry uri, fetch the set uri, and compare
+        # it with the tracking uri. They should be different
+        mlflow.set_registry_uri("sqlite:///tmp/registry.db")
+        mr_uri = mlflow.get_registry_uri()
+        print("Current registry uri: {}".format(mr_uri))
+        tracking_uri = mlflow.get_tracking_uri()
+        print("Current tracking uri: {}".format(tracking_uri))
+
+        # They should be different
+        assert tracking_uri != mr_uri
+
+    .. code-block:: text
+        :caption: Output
+
+        Current registry uri: sqlite:///tmp/registry.db
+        Current tracking uri: file:///.../apis/mlruns
```

Comment:
```suggestion
        Current tracking uri: file:///.../mlruns
```

Can we remove `apis`?

Reply:
No, the actual paths being printed are:

```
Current registry uri=sqlite:////tmp/registry.db
Current tracking uri: file:///Users/julesdamji/examples/py/mlflow/apis/fluent/mlruns
```

I just `...`. Perhaps better to only have the ```file:///.../mlruns``` across all